freshmaker/pyxis.py

```diff
@@ -215,6 +215,135 @@ def get_images_by_digests(self, digests):
         :return: bundle ContainerImages
         :rtype: list
         """
-        request_params = {'include': 'data.brew',
-                          'filter': f"repositories.manifest_list_digest=in=({','.join(digests)})"}
-        return self._pagination("images", request_params)
+        request_params = {'include': 'data.brew,data.repositories',
+                          'filter': f'repositories.manifest_list_digest=in=({",".join(digests)})'}
+        return self._pagination('images', request_params)
+
+    def _add_repositories_info(self, reg_repo_info):
+        """
+        For every pair of registry-repository add information about it's
+        auto_rebuild tags. To decrease amount of queries to Pyxis, only one
+        query is performed with the filter set to proper registry-repository
+        pair.
+
+        Tags list is added to the parameter object of the method.
```

Comment:
Could you please reword this to be clearer? How about?
```suggestion
        A list of tags for each repository will be added to the 'tags' key in the input info about the repository.
```